worlds/openrct2/__init__.py

```diff
@@ -0,0 +1,590 @@
+import math
+from typing import TextIO
+from Utils import local_path
+
+import worlds.LauncherComponents as LauncherComponents
+from BaseClasses import ItemClassification, Region, Location, Tutorial
+from worlds.generic.Rules import add_rule
+
+from .Constants import base_id, apworld_version
+from .data.scenario_info import scenario_info
+from .data.item_info import item_info
+from .data.location_info import location_info
+from .Items import OpenRCT2Item, set_openRCT2_items
+from .Options import openRCT2Options, Scenario, openrct2_option_groups
+from worlds.AutoWorld import World, WebWorld
+
+
+class OpenRCT2WebWorld(WebWorld):
+    theme = "partyTime"
+
+    setup_en = Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up the OpenRCT2 randomizer on your computer.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["Crazycolbster"]
+    )
+
+    tutorials = [setup_en]
+    option_groups = openrct2_option_groups
+
+
+class OpenRCT2Location(Location):
+    game = "OpenRCT2"
+
+
+def launch_client() -> None:  # Shoutout to Serpent.ai for the launcher code!
+    from .Client import main
+    LauncherComponents.launch_subprocess(main, name="OpenRCT2Client")
+
+
+LauncherComponents.components.append(
+    LauncherComponents.Component(
+        "OpenRCT2 Client",
+        func=launch_client,
+        component_type=LauncherComponents.Type.CLIENT,
+        # OpenRCT2 icon credit to the OpenRCT2 team: 
+        # https://github.com/OpenRCT2/OpenRCT2/blob/develop/resources/logo/icon_x96.png
+        icon='openrct2icon' 
+    )
+)
+
+
+LauncherComponents.icon_paths['openrct2icon'] = local_path('data', 'openrct2icon.png')
```

Comment:
```suggestion
LauncherComponents.icon_paths['openrct2icon'] = f"ap:{__name__}/icons/openrct2icon.png"
```

And then move the icon from the root `data` folder to a `worlds/openrct2/icons` folder. This will take advantage of the newer icon API.

Reply:
You should never apologize for improving a project!